frontend/src/components/Login/Login.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { FormControl, TextField, Button, Typography } from '@mui/material';
+import {
+  BorderForm,
+  RegistrationFormWrapper,
+  StyledSpan,
+  StyledSpanEnd
+} from 'components/Login/styles';
+import {
+  TextFieldWrap,
+  WrapButtonAndText,
+  WrapH1
+} from 'components/ForgotPassword/styles';
+
+function Login() {
+  return (
+    <RegistrationFormWrapper>
+      <BorderForm>
+        <FormControl sx={{ width: '35ch' }}>
+          <WrapH1>
+            <Typography>Login</Typography>
+          </WrapH1>
+          <TextFieldWrap>
+            <TextField
+              placeholder="Please enter your email"
+              label="Email"
+              autoComplete="current-email"
+            />
+          </TextFieldWrap>
+
+          <TextFieldWrap>
+            <TextField
+              placeholder="Please enter your email"
+              label="Password"
+              type="password"
+              autoComplete="current-password"
+            />
+          </TextFieldWrap>
+
+          <StyledSpanEnd>
+            <span>Forgot the password?</span>

```

Comment:
also should be a link to forgotPassword page

Reply:
done